### ISSUE TITLE
I want to get a PFS automatically selected and paid

### ISSUE THREAD

**Issue by christianbrb** [raiden-network/light-client#639]
## User Story
As Chris I want to get a PFS automatically selected  and paid so that I do not have to think about it on my own.

**Comment by andrevmatos** [comment]
I think after #600 , if the PFS is free, it's already auto-selected and even dialog is skipped.
If it's paid, PFS selection also wraps user accepting its fee and paying for it, so I'd say we should always show this confirmation dialog. Nonetheless, the "best" PFS is always pre-selected, so the user could just confirm and move on. As it may cost user money, I'm not sure we should automate more than that.

**Comment by christianbrb** [comment]
We have estimated 40 for #639 and #641 in combination. I am putting 20 on every individual story.

**Comment by christianbrb** [comment]
Issue housekeeping looking at least recently updated issues.

Nice to have. If the request is going to be raised by a user we'll reopen the issues.
